src/components/NavFooter.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FaGithub } from 'react-icons/fa'; 
```

Comment:
Build failed again probably because you're importing this, but `react-icons` doesn't exist in `package.json`
You'll need to install it. More details here - https://react-icons.github.io/react-icons/
Check the **Installation (for standard modern project)** section